athena-mysql/README.md

```diff
@@ -0,0 +1,177 @@
+# Amazon Athena Lambda MySQL Connector
+
+This connector enables Amazon Athena to access your MySQL database or RDS instance(s). 
```

Comment:
the ` or RDS instance(s). ` made sense when it was single connector. I don't think we need it anymore

Reply:
same for postgres readme